feat: add "Busy" status and new preset "Be right back" status

miaulalala opened the pull request:
## Summary

Implements changes as discussed in https://github.com/nextcloud/spreed/issues/15465

Close https://github.com/nextcloud/server/pull/54177

## TODO

- [ ] Vue Library update
- [x] frontender work to align modal

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required

miaulalala committed:
feat: add Busy status and new preset BRB status

Signed-off-by: Anna Larch <anna@nextcloud.com>

nickvergessen reviewed (approved):
Looks good from technical side. The frontend needs some polishing thou 🙈

Antreesy committed:
style(user_status): adjust status modal styles

- remove modal name
- propagate box-sizing
- reduce gap between sections 12px -> 8px
- align icons vertically
- add hover effect to status labels

Signed-off-by: Maksim Sukharev <antreesy.web@gmail.com>

st3iny reviewed (approved):
LGTM!

Antreesy commented:
/compile

nextcloud-command committed:
chore(assets): Recompile assets

Signed-off-by: nextcloud-command <nextcloud-command@users.noreply.github.com>
